diff -up ypbind-mt-1.37.2/configure.in.systemdso ypbind-mt-1.37.2/configure.in
--- ypbind-mt-1.37.2/configure.in.systemdso	2014-08-15 10:47:04.183182884 +0200
+++ ypbind-mt-1.37.2/configure.in	2014-08-15 10:47:37.182232398 +0200
@@ -105,10 +105,10 @@ AM_CONDITIONAL(ENABLE_REGENERATE_MAN, te
 
 USE_SD_NOTIFY=0
 AC_SUBST(USE_SD_NOTIFY)
-AC_CHECK_LIB(systemd-daemon,sd_notify,LIBSYSTEMD_DAEMON="-lsystemd-daemon",
+AC_CHECK_LIB(systemd,sd_notify,LIBSYSTEMD_DAEMON="-lsystemd",
        LIBSYSTEMD_DAEMON="")
 if test -z "$LIBSYSTEMD_DAEMON" ; then
-   AC_CHECK_LIB(systemd,sd_notify,LIBSYSTEMD_DAEMON="-lsystemd",
+   AC_CHECK_LIB(systemd-daemon,sd_notify,LIBSYSTEMD_DAEMON="-lsystemd-daemon",
    LIBSYSTEMD_DAEMON="")
 fi
 if test -n "$LIBSYSTEMD_DAEMON" ; then
