Index: src/serv_list.c
===================================================================
--- src/serv_list.c
+++ src/serv_list.c	2009-11-30 08:54:45.000000000 +0100
@@ -629,6 +629,11 @@
           return 0;
         }
 
+      memcpy(&(in_use->server[0].addr), &addr->sin_addr,
+            sizeof (struct in_addr));
+      memcpy(&(in_use->server[0].port), &addr->sin_port,
+            sizeof (unsigned short int));
+
       memcpy (&(in_use->sin), &addr, sizeof (struct sockaddr_in));
 
       in_use->active = 0;
