--- ypbind-mt-1.32/src/Makefile.am.bero	2013-02-08 14:18:11.928255220 +0100
+++ ypbind-mt-1.32/src/Makefile.am	2013-02-08 14:18:17.491190892 +0100
@@ -9,7 +9,7 @@ localedir = $(datadir)/locale
 WARNFLAGS = @WARNFLAGS@
 AM_CFLAGS = -D_REENTRANT=1 $(WARNFLAGS) -DUSE_BROADCAST=@USE_BROADCAST@ \
 	-DLOCALEDIR=\"$(localedir)\" @DBUS_CFLAGS@ @GLIB_CFLAGS@
-ypbind_LDADD = ../lib/libcompat.a @LIBINTL@ @SLP_LIBS@ @DBUS_LIBS@ @GLIB_LIBS@
+ypbind_LDADD = ../lib/libcompat.a @LIBINTL@ @SLP_LIBS@ @DBUS_LIBS@ @GLIB_LIBS@ -ltirpc
 
 CLEANFILES = *~
 
